README.md

```diff
@@ -87,4 +87,5 @@ Here's is a list of the things that happen when `now-lambda` process your `now.j
 ## CLI arguments
 
 * `--config` - path to `now.json` file
-* `--port` - by default the local server listens on port 8004. You can change it via this argument.
\ No newline at end of file
+* `--port` - by default the local server listens on port 8004. You can change it via this argument.
+* `--proxy` - a URL to use as a fallback proxy for requests that would otherwise fail (e.g. `http://localhost:3000`)
```

Comment:
Can you please add one sentence saying that this is not a standard `now`/zeit feature. It is built here for convenience.

Reply:
Thanks for taking a look, will do!

Will add to the read README - let me know if you like the inline help to be more explicit too (was torn between keeping it short and explaining a bit more).

Reply:
I think it's fine.